spec/requests/new_document_spec.rb

```diff
@@ -4,9 +4,10 @@
   describe "GET /documents/choose-document-type" do
     it "shows supertype when a supertype managed by content publisher is selected" do
       supertype = Supertype.all.reject(&:managed_elsewhere).first
-      get choose_document_type_path, params: { supertype: supertype.id }
+      supertype_id = supertype.id
+      get choose_document_type_path, params: { supertype: supertype_id }
       expect(response).to have_http_status(:ok)
-      expect(response.body).to have_content(supertype.label)
+      expect(response.body).to have_content(I18n.t("supertypes.#{supertype_id}.label"))
```

Comment:
We should use `I18n.t!` in tests, as the body is quite likely to have an empty string, which is the `t` fallback.

Reply:
thanks :)